Hello world feature

Freherdez opened the pull request:
In this pull we:

Added .gitignore file
Started setting up a linter for HTML and CSS
Added .github folder
Added workflows file
Added HTML file
Added CSS file
Added test.md

Freherdez committed:
Create .gitignore

Added gitignore

Freherdez committed:
Create workflows

Freherdez committed:
Update README.md

Freherdez committed:
Update README.md

Freherdez committed:
Added html file

Freherdez committed:
Create styles.css

Freherdez committed:
Update and rename .html to index.html

rename html document and update to set CSS link

Freherdez committed:
Create test.md

Added test.md

Freherdez committed:
Added test.md

osamaashraf6 commented on test.md:
```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```
- [ ] Great job 👌at adding the ``` test.md ``` file at the gitignore, but there are some issues 😃you need to work I see that you already put the ``` test.md``` file at the gitignore, but it is still existing at the repo on GitHub😊, so kindly remove the ``` test.md``` file from the remote repo as it is from the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)💯

osamaashraf6 commented on .github/workflows:
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
```
- [ ] You did well 😃at adding the linter files and folders, but I see that you forgot something to do😊 , I think you forgot to add inside the workflows the ``` linters.yml``` file, so the linter configuration will work the directories of linter configuration like that ``` .github/workflows/linters.yml```💯you can read about linter [Here](https://github.com/microverseinc/linters-config/tree/master/html-css-js)⤵️

Freherdez replied:
@osamaashraf6 hi, thanks for the feedback I really appreciate it, I have little doubt, I need only set de file linters.yml to the workflow folder. because when I click on the link I can see all configurations, or set the link and after that set all configurations to linters.yml?

osamaashraf6 commented on README.md:
```diff
@@ -1 +1,2 @@
-# Hello-world
\ No newline at end of file
+![microverselogo](https://user-images.githubusercontent.com/109646457/211634493-6e518a79-9005-4816-80da-84bf3a14fb6a.png)
+                                                               Hello Microverse Project
```
- [ ] Kindly 😊add to your README file, the [template provided by Microverse](https://github.com/microverseinc/readme-template)⤵️ and complete it with your personal information💯. In this way, you will have documented your work in a professional way😃

osamaashraf6 reviewed (changes requested):
Hi @Freherdez 👋, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

![SoCloseAlmostGIF](https://user-images.githubusercontent.com/83360636/211644747-5c7dcf67-7551-4ed5-8127-3d3a666b79e2.gif)


## Highlights ✔️
- [x] Good commit messages.
- [x] Good usage in correcting GitHub flow💯 
- [x] Descriptive pull request


## Required Changes ♻️

- [x] Good job👍with the project you are almost there you finish the project don't worry 🥳your code is good, but there are some issues to be fixed, so please check the comments to know what I am talking about 😊. 

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

Freherdez committed:
Delete test.md

Delete test.md from the project requirements.

Freherdez committed:
Update README.md

Freherdez committed:
Added microverse log

Freherdez committed:
Update logo Readme.md

Freherdez committed:
Delete workflows to set linter file

Freherdez committed:
New workflows floder with linter file

Freherdez committed:
Update styles.css

osamaashraf6 commented on README.md:
```diff
@@ -1 +1,127 @@
-# Hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```
Great job 😃 at adding the changes ♻️and updating some sections from the README file☺️, but I see you use it  , but you didn't update somethings as stated by the previous reviewer, kindly update the README file here it is not necessary to leave these comments in the file💯, so please it will be better if you remove the comments.

osamaashraf6 commented on README.md:
```diff
@@ -1 +1,127 @@
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse" project <a name="about-project"></a>
+
+> Say Hello.
+
+**Hello World** is a...
+
```
- [ ] Kindly😃 put here some sentences about what this project do, so the users when seeing this project will understand 👌what this project about💯

osamaashraf6 commented on README.md:
```diff
@@ -1 +1,127 @@
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
- [ ] Great job 👌at updating the README file, but I see that the MIT LICENCE file is broken☺️, so kindly try to fix the LICENCE file by adding it from GitHub💯

osamaashraf6 commented on README.md:
```diff
@@ -1 +1,127 @@
+
+👤 **Author1**
+
+- GitHub: [@Freherdez)
```
- [ ] Great job 😃 at adding your information here, but when the user clicks on your name it will not redirect 👌him/her to your GitHub account, so kindly😊as stated by the previous reviewer update and  try to put your GitHub account like this
 ```[Here type your name](Here put the link of your GitHub account)```, so the user when clicks on the link it will redirect to your account and see your projects😊💯

osamaashraf6 reviewed (changes requested):
Hi @Freherdez  ☺️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

![ItsJustATinyChangeSheaWhitneyGIF (2)](https://user-images.githubusercontent.com/83360636/211677210-e7098572-f01d-48db-b484-5351a628903f.gif)


## Highlights ✔️
- [x] Good commit messages.
- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow


 ## Required Changes ♻️

- [ ] Great job😃 I see that you used the Microverse Readme template as stated by the previous reviewer👌, but I see that you removed some sections from it like the future features and others💯, so Kindly make sure all the main sections are present in yours and placeholders are replaced with your project and personal data. The only optional section is a FAQ. Please remove the template logo on the top, if you have one, you can replace it. This is crucial to have a professional and instructive repository. 👌

_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Freherdez committed:
Update README.md

Add extra information on README.md, fix links Authors section, and Delete all useless comments on the document.

Freherdez committed:
Add MIT.md file

Freherdez committed:
Fix MIT link on section License

Freherdez committed:
Update MIT.md

algerina reviewed (approved):
Hi @Freherdez  ,
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:

<img src="https://media2.giphy.com/media/artj92V8o75VPL7AeQ/giphy.gif"/>

 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_
